turnaroundmarker:
  defaults:
    target_yaw: 90
    timeout: 10

movepastmarker:
  defaults:
    target_surge_time: 10

turntogate:
  defaults:
    target_yaw: 180
    timeout: 10

returntogate:
  defaults:
    target_surge_time: 26
